armi/cases/tests/test_suiteBuilder.py

```diff
@@ -46,19 +46,20 @@ def __call__(self, cs, bp, geom):
 
 
 class TestLatinHyperCubeSuiteBuilder(unittest.TestCase):
-    """
-    Class to test LatinHyperCubeSuiteBuilder.
-
-    .. test:: A generic mechanism to allow users to modify user inputs in cases.
-        :id: T_ARMI_CASE_MOD0
-        :tests: R_ARMI_CASE_MOD
-    """
+    """Class to test LatinHyperCubeSuiteBuilder."""
 
     def test_initialize(self):
         builder = LatinHyperCubeSuiteBuilder(case, size=20)
         assert builder.modifierSets == []
 
     def test_buildSuite(self):
+        """
+        Initialize an LHC suite.
+
+        .. test:: A generic mechanism to allow users to modify user inputs in cases.
+            :id: T_ARMI_CASE_MOD0
+            :tests: R_ARMI_CASE_MOD
+        """
```

Comment:
I have no issue with this test tag being moved here, but I wanted to comment that I think we're not testing the right thing in general here. 

Nobody uses this latin hypercube sampler, to my knowledge. We are more typically using the `FullFactorialSuiteBuilder` or `SeparateEffectsSuiteBuilder` classes. It would seem that we should instead be tagging tests for those, though I don't seem to see any tests for those classes.

So maybe we need to add some tests, and then also tag those new tests to satisfy this R_ARMI_CASE_MOD requirement.

Let me know if that's unclear.